js/compressed-token/src/actions/transfer.ts

```diff
@@ -101,56 +58,70 @@ export async function transfer(
         },
     );
 
-    const [inputAccounts, inputAmount, inputLamports] =
-        selectMinCompressedTokenAccountsForTransfer(
-            compressedTokenAccounts,
-            amount,
-        );
-
-    /// TODO: refactor into createOutputState
-    /// Create output compressed accounts
-    const changeAmount = inputAmount.sub(amount);
-    /// We don't send lamports and don't have rent
-    const changeLamportsAmount = inputLamports;
-
-    const changeCompressedAccount: TokenTransferOutputData = {
-        amount: changeAmount,
-        owner: currentOwnerPublicKey,
-        lamports: changeLamportsAmount,
-    };
-
-    const recipientCompressedAccount: TokenTransferOutputData = {
+    const [inputAccounts] = selectMinCompressedTokenAccountsForTransfer(
+        compressedTokenAccounts,
         amount,
-        owner: toAddress,
-        lamports: null,
-    };
+    );
 
     const proof = await rpc.getValidityProof(
         inputAccounts.map(account => bn(account.compressedAccount.hash)),
     );
 
-    const ixs = await createTransferInstruction(
-        payer.publicKey, // fee payer
-        currentOwnerPublicKey, // authority
-        inputAccounts.map(
-            account => account.compressedAccount.merkleTree, // in state trees
-        ),
-        inputAccounts.map(
-            account => account.compressedAccount.nullifierQueue, // in nullifier queues
-        ),
-        [merkleTree, merkleTree], // out state trees
-        inputAccounts.map(account => account.compressedAccount), // input compressed accounts
-        [recipientCompressedAccount, changeCompressedAccount], // output compressed accounts
-        // TODO: replace with actual recent state root index!
-        // This will only work with sequential state updates and no cranking!
-        proof.rootIndices, // input state root indices
-        proof.compressedProof,
-    );
+    const ixs = await CompressedTokenProgram.transfer({
```

Comment:
Naming is confusing to me ``createTransferInstruction`` is more intuitive in my opinion.
The name transfer leads me to expect for a transaction being sent not instructions being returned.

Reply:
yea agree it's more descriptive however .transfer() matches the solana systemprogram implementation, so there's familiarity for folks who work with it e.g. in browser environments...